src/components/Subscribe/Subscribe.tsx

```diff
@@ -63,11 +102,19 @@ const Subscribe = ({
     >
       <h3 className={`${px}-subscribe__title`}>{title}</h3>
       {blurb ? <p className={`${px}-subscribe__blurb`}>{blurb}</p> : null}
+
       <Input
         className={`${px}-subscribe__input`}
         type="email"
+        name="email"
         placeholder={inputPlaceholder}
         labelText={inputLabelText}
+        invalid={isInvalid}
+        invalidText={invalidText}
+        warn={warn}
```

Comment:
I don't see any warn UX in the spec for the Input component here: https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5730-11728&t=Nl1MOE1dSPwIxfrS-4